### PR TITLE
Setup end to end testing with Cypress

### LINKED ISSUE #16: Setup End to End Testing
- [x] choose the tool for e2e testing: *Cypress*
- [x] install and configure Cypress
- [x] create a first passing test

### PR DESCRIPTION
**Pull request by riccardo-giomi**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -21,7 +22,7 @@
     "ecmaVersion": 11,
     "sourceType": "module"
   },
-  "plugins": ["react", "react-hooks", "prettier"],
+  "plugins": ["react", "react-hooks", "prettier", "cypress"],
```
Comment (zsid) [inline .eslintrc.json]: Add `prettier` as the last plugin. It was recommended in their docs to be last for formatting to work as expected

**Review by zsid — APPROVED** [review]
Looks good. 1 small comment for the order

**Review by riccardo-giomi — COMMENTED** [review]
Change the plugins order in the .eslitrc.js file, prettier is now last.

File: .eslintrc.json (hunk 1 above)
Comment (riccardo-giomi) [reply .eslintrc.json]: Thanks for the tip, I changed it.

**Pull request merged**